samples/CommunityToolkit.Maui.Sample/Pages/Converters/CompareConverterPage.xaml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
I've become so used to seeing this in XAML files I've never considered checking whether it needs to be there or not. My google skills are not helping me here, are we content that the files are going retain the utf8 encoding without this? Does it matter?

Reply:
I don't really know if there would be implications here, I guess I have never questioned it's existence before 😄.

I'm all for consistency. Let's keep it out for now.